### ISSUE TITLE
Cannot create Tunisia CoA

### ISSUE THREAD

**Issue by Nofal-Mami** [frappe/books#512]
Hi,
I have problem with this part
![image](https://user-images.githubusercontent.com/35137381/207569965-c232f127-a85c-4719-bd9c-fcff9e24737a.png)

since Tunisia COA is build with each account have his type, means you can find a group of accounts having asset and liability account types in the same group.

Can you suggest a solution for that.

**Comment by 18alantom** [comment]
As of now, this is not possible. I'll have to update the software to have an `isCredit` flag.

Can you contribute the Tunisian CoA, where the accounts have an `isCredit` flag?

I can update the software once we have that.

For reference, here's the Indian CoA: [link](https://github.com/frappe/books/blob/master/fixtures/verified/in.json#L13), notice the `isGroup` flag. The `isCredit` flag will be similar to this for all the child accounts.

**Comment by Nofal-Mami** [comment]
so if isCredit checked that means this account is credit type even it belongs to a debit group type am I correct?
second if not checked that means this acciunt is debit type even it belongs to credit group type.
By default if parent is credit type, isCredit will be checked, same if parent is debit isCredit will be unchecked right?
now in reporting, how balance sheet will behave?

**Comment by 18alantom** [comment]
> so if isCredit checked that means this account is credit type even it belongs to a debit group type am I correct?

Correct

> second if not checked that means this acciunt is debit type even it belongs to credit group type. By default if parent is credit type, isCredit will be checked, same if parent is debit isCredit will be unchecked right?

Correct

> how balance sheet will behave?

Here the accounts will be grouped according to the `isCredit` flag.

And for the parent, something like `"(Credit)"` or `"(Debit)"` will be used to show nature of the accounts underneath it. This way the parent can be displayed on both the Credit and the Debit side depending on the child account's `isCredit` flag.

**Comment by Nofal-Mami** [comment]
Agree, even we need to do same in ERPNext couls share this approach to ERPNext accounting team, many countries want to use erpnext but due to this issue they are blocked, you can check in discuss.erpnext.com how many thread on this topic.

**Comment by Nofal-Mami** [comment]
[Plan comptable STE COOPERATION.xls](https://github.com/frappe/books/files/10228186/Plan.comptable.STE.COOPERATION.xls)
attached Tunisian COA, parent child in the same column but parent code is 10, child 101, next child is 10100000, and same for others, account type O: owner equity, L:liability, E:Expense, R:Revenu, A:Asset

**Comment by Nofal-Mami** [comment]
check for accounts starting by 4... tiy will find L and A

**Comment by 18alantom** [comment]
Thanks, I'm finding the nesting of accounts a bit confusing. Will take a proper look.

Here I'm assuming, the `isCredit` flag will be set like so:

| Accounts                                     | `isCredit` |
| -------------------------------------------- | ---------- |
| **L**iability, **R**evenue, **O**wner Equity | `true`     |
| **A**ssets, **E**xpense                      | `false`    |

**Comment by Nofal-Mami** [comment]
yes it's confusing, actually account grouping is not following root type like Liability, Revenue, Owner Equity ... it's different.
your approach is correct.

**Comment by Gadha2311** [comment]
Could you please confirm if you are still encountering this issue, or has it been resolved on your end?

**Comment by Nofal-Mami** [comment]
sorry did not checked since that day. No more using books. is theire any development for this issue implemented so I can check again?
